test/binary/bad-memory-limits-flag.txt

```diff
@@ -3,9 +3,9 @@ magic
 version
 section(MEMORY) {
   count[1]
-  flags[4]
+  flags[8]
```

Comment:
These tests were here on purpose. It is surprising that you are modifying them without any considerations.
Fixed by https://github.com/WebAssembly/wabt/pull/1547.

Reply:
Unfortunately, the way testing is done in wabt does make this more likely to do -- it's very easy to rebase the tests and accidentally hide a bug.

Reply:
No, I changed these by hand, and I did consider it.. I just somehow thought it was the right thing to do.